app/models/concerns/wp_cache.rb

```diff
@@ -23,17 +23,18 @@ def classes
     #
     # Schedules a `create_or_update` call to itself.
     #
-    def schedule_create_or_update(wp_id)
-      WpApiWorker.perform_async(self, wp_id)
+    def schedule_create_or_update(wp_id, preview=false)
+      WpApiWorker.perform_async(self, wp_id, preview)
     end
 
     #
     # Gets the content from the WP API, finds-or-creates a record for it,
     # and passes it the content by the `update_wp_cache` instance method.
     #
-    def create_or_update(wp_type, wp_id)
+    def create_or_update(wp_type, wp_id, preview=false)
       return unless wp_id.is_a? Fixnum or wp_id.is_a? String
-      wp_json = get_from_wp_api "#{ wp_type }/#{ wp_id }"
+      maybe_preview_segment = (preview ? "preview/" : "")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.